kubetest2/pkg/types/types.go

```diff
@@ -79,6 +79,17 @@ type Deployer interface {
 	Build() error
 }
 
+// Some testers may need information about the deployer not available in the standard
+// deployer interface.
+
+// GinkgoableDeployer adds a few methods needed for ginkgo tests.
+type GinkgoableDeployer interface {
+	Deployer
+
+	Kubeconfig() (string, error)
```

Comment:
we should clarify if this returns the contents or file path

Reply:
Done.